hack/make/tgz

```diff
@@ -19,15 +19,29 @@ for d in "$CROSS/"*/*; do
 	mkdir -p "$DEST/$GOOS/$GOARCH"
 	TGZ="$DEST/$GOOS/$GOARCH/$BINARY_NAME.tgz"
 
-	mkdir -p "$DEST/build/usr/local/bin"
-	cp -L "$d/$BINARY_FULLNAME" "$DEST/build/usr/local/bin/docker$BINARY_EXTENSION"
-	copy_containerd "$DEST/build/usr/local/bin/"
+	# The staging directory for the files in the tgz
+	BUILD_PATH="$DEST/build"
 
-	tar --numeric-owner --owner 0 -C "$DEST/build" -czf "$TGZ" usr
+	# The directory that is at the root of the tar file
+	TAR_BASE_DIRECTORY="docker"
```

Comment:
If we decide we don't want it to be docker, we just need to change this one variable to what ever we want. If we don't want any directory, we will need to leave this blank, or even better just remove the variable all together, since it won't be needed anymore.